bin/cli.rs

```diff
@@ -0,0 +1,126 @@
+#![allow(missing_docs)]
+use std::{
+    collections::BTreeMap,
+    fs::{create_dir_all, File},
+    io::BufReader,
+    num::NonZeroUsize,
+    thread,
+};
+
+use alloy_consensus::constants::KECCAK_EMPTY;
+use alloy_primitives::{Address, B256};
+use alloy_provider::{Provider, ProviderBuilder};
+use alloy_rpc_types::{BlockId, BlockTransactionsKind};
+use clap::{Parser, Subcommand};
+use pevm::{
+    chain::{PevmChain, PevmEthereum},
+    EvmAccount, EvmCode, Pevm, RpcStorage,
+};
+use reqwest::Url;
+use tokio::runtime;
+
+#[derive(Parser, Debug)]
+pub struct Cli {
```

Comment:
@hai-rise, naming is indeed hard here. I was thinking that this CLI could be extended in the future to provide further functionality, e.g. could spin up a local anvil instance to run in CI.

Would appreciate any thoughts on on naming here. Thank you.

Reply:
Yeah, I think a `pevm` CLI would be sick. For now, though, let's keep it simple by naming it something like `fetch` without any sub-command.